service/src/main/java/bio/terra/drshub/services/DrsResolutionService.java

```diff
@@ -149,11 +149,6 @@ private DrsMetadata fetchObject(
     var accessMethod = AccessMethodUtils.getAccessMethod(drsResponse, drsProvider, cloudPlatform);
     var accessMethodType = accessMethod.map(AccessMethod::getType).orElse(null);
 
-    if (drsProvider.shouldFetchUserServiceAccount(accessMethodType, requestedFields)) {
```

Comment:
Can the shouldFetchUserServiceAccount method also be removed from DrsProviderInterface?

Reply:
eventually, yes. There is a bunch more to remove too.